containers/podman-desktop
# feat: Adds Kind extension

### What does this PR do?
Allow to call kind tool and
- list kind clusters
- Allows to create clusters

draft as it requires all my PR being merged first

### Screenshot/screencast of this PR

https://user-images.githubusercontent.com/436777/208451335-fc134ad7-21fb-4011-a632-cc681d5787ee.mp4



### What issues does this PR fix or reference?

Fixes #413 

### How to test this PR?

Go to settings/resources/Kind menu
- if you have some kind clusters you should see them
- you should be able to create one cluster

Change-Id: Ib9174b6a453d1970fe4bec54fb29013ebd5db436
Signed-off-by: Florent Benoit <fbenoit@redhat.com>
